lib/announce/adapters/shoryuken_adapter.rb

```diff
@@ -33,6 +33,7 @@ def subscribe(worker_class, subject, actions, options)
         def register_class(worker_class)
           if active_job?
             Class.new(AnnounceWorker).tap do |jc|
+              jc.class_eval("shoryuken_options body_parser: :json, auto_delete: true")
```

Comment:
I haven't seen a problem with that, but it may be b/c my projects define global config for shoryuken in an initializer.